cli/internal/plugin/builtin/oci/plugin.go

```diff
@@ -151,7 +155,7 @@ func (p *Plugin) createRepository(spec *ociv1.Repository, credentials map[string
 		oci.WithResolver(urlResolver),
 		oci.WithScheme(p.scheme),
 		oci.WithCreator(Creator),
-		oci.WithOCIDescriptorCache(p.memory),
+		oci.WithManifestCache(p.memory),
```

Comment:
I think we need a layer cache, I probably missed this the first time (and 2 memory structs)